linchpin/shell/__init__.py

```diff
@@ -267,8 +278,10 @@ def up(ctx, targets, run_id, tx_id, inventory_format):
 @click.option('-t', '--tx-id', metavar='tx_id', type=int,
               help='Destroy resources using the transaction ID (tx-id)',
               cls=MutuallyExclusiveOption, mutually_exclusive=["run_id"])
+@click.option('--rhof', '--run-hooks-on-failure', is_flag=True)
+@click.option('-nh', '--no-hooks', is_flag=True)
```

Comment:
Again, you need to have two dashes (`--nh`) to conform to the linchpin standard options.

Reply:
@samvarankashyap I updated your code with the `--nh` and `--ifh` and they work just fine. I'm not sure why they don't work for you. Please change this back.